README.md

```diff
@@ -524,30 +543,71 @@ backend {
     // This is the maximum polling interval (in seconds):
     maximumPollingInterval = 600
   }
+}
+```
 
-  ...
+### Configuring Authentication
+
+The `google` stanza in the Cromwell configuration file defines how to authenticate to Google.  There are three authentication schemes:
+
+* `application_default` - (default, recommended) Use [application default](https://developers.google.com/identity/protocols/application-default-credentials) credentials.
+* `service_account` - Use a specific service account and key file (in PEM format) to authenticate.
+* `user_account` - Authenticate as a user.
+
+#### Application Default Credentials
+
+By default, application default credentials will be used.  The configuration file should look like this to use application default credentials:
+
+```hocon
+google {
+  applicationName = "cromwell"
+  cromwellAuthenticationScheme = "application_default"
 }
 ```
 
-It is also necessary to fill out the `google` stanza in the configuration file. This stanza will set up the service / user account that Cromwell uses to write certain files to GCS as well as run jobs.
+To authenticate, run the following commands:
+
+```
+$ gcloud auth login
+$ gcloud config set project my-project
+```
+
+This should be all that's necessary to run Cromwell using the JES backend.
+
+#### Service Account
```

Comment:
No you don't, when you gcloud auth login you login using a gmail account basically. You can still [use gcloud to authenticate using a service account](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account) but you don't have too